course_catalog/api.py

```diff
@@ -112,7 +112,9 @@ def digest_ocw_course(
                     "description": master_json.get("image_description"),
                 },
                 "max_modified": last_modified,
-                "content_type": ContentType.objects.get(model="course").id,
+                "content_type": ContentType.objects.get(
+                    model="course", app_label="course_catalog"
+                ).id,
```

Comment:
Made this change in course_catalog/search/infrastructure apps just to keep tests passing.